plaidml/bridge/openvino/tests/functional/plaidml/single_layer_tests/BUILD

```diff
@@ -10,7 +10,7 @@ TAGS = [
 
 cc_test(
     name = "single_layer_tests",
-    srcs = glob(["*.cpp"]),
+    srcs = glob(["transpose.cpp"]),
```

Comment:
Leave this as `"*.cpp"`

Reply:
oops. sorry.

Reply:
BTW, if you want to run a single test, look at the docs for gtest here: 
https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#running-a-subset-of-the-tests

basically:

```
bazelisk run //some/test -- --gtest_filter=Fixture.Case
```